bool sync
float64 timeAcq
int64 sensLeft
int64 sensRight
int64 posLeft
int64 posRight